.github/workflows/ci.yml

```
@@ -3,8 +3,8 @@ name: CI
 on:
   push:
     paths:
-      - .github/workflows/ci.yml
-      - pkg/**
+      - '.github/workflows/*.yml'
```

you might want to add one more `*` 🙂 
```suggestion
      - '.github/workflows/**.yml'
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths

> The * wildcard matches any character, but does not match slash (/).